str/runners/str_iterative_eh_runner.py

```diff
@@ -52,105 +59,116 @@ def main(
     )
     b = hb.Batch(backend=backend, default_image=os.getenv('DRIVER_IMAGE'))
```

Comment:
```suggestion
    b = get_batch()
```
Please remove this constructor and just use `cpg_utils.hail_batch.get_batch`